### DISCUSSION TITLE
unraid虚拟机安装报错，大佬能帮忙看看吗？

### DISCUSSION THREAD

**Discussion by shinelogo** [kkkgo/PaoPaoGateWay#118] [问和答] [answered]
DNS地址是192.168.2.2
网关地址是192.168.2.3
<img width="1606" alt="Snipaste_2024-09-01_01-01-29" src="https://github.com/user-attachments/assets/7af8d7cc-d232-4ba9-b16f-38bdd2e1f509">

**Comment by kkkgo** [comment, marked as answer]
你这截图一个error的单词都没有，哪里报错了。

**Reply by shinelogo** [reply]
呃呃，我看到ICPMv6那个类似错误的提示之后，下面的所有提示都是各种try，一只重复

**Reply by kkkgo** [reply]
try就是不停尝试获取刷新配置啊，配置变了就同步应用配置。

**Reply by shinelogo** [reply]
<img width="1138" alt="image" src="https://github.com/user-attachments/assets/56090fa0-6186-4fcf-a09b-25e954824a88">
free模式这个报错怎么解决？

**Reply by kkkgo** [reply]
这解决啥，你开了fast_node=yes，这说明你的free模式访问不了test_node_url

**Reply by shinelogo** [reply]
明白了，我现在应该换成yaml模式，把节点配上去就可以访问UI对吗？

**Reply by kkkgo** [reply]
看文档，free模式是你有物理专线的情况下用的。
